docs/manifest.json

```diff
@@ -69,6 +69,11 @@
           "icon_path": "./images/icons/docker.svg",
           "path": "./quickstart/docker.md"
         },
+        {
+          "title": "Google Cloud Platform",
+          "description": "Setup Coder with Google Cloud Platform",
+          "path": "./quickstart/google-cloud-platform.md"
```

Comment:
Let's add an icon to this PR. icons8 can be used to find a nice SVG

Reply:
done